crates/config/src/etherscan.rs

```diff
@@ -251,18 +263,26 @@ pub struct ResolvedEtherscanConfig {
     pub browser_url: Option<String>,
     /// The resolved API key.
     pub key: String,
+    /// Etherscan API Version.
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub api_version: Option<String>,
```

Comment:
should this also be just etherscanapiversion here?

Reply:
I wasn’t too sure what the pattern was for the __Resolved struct if it was recommended to be a string since the other fields were and it’d be parsed later. If this is the case I can switch it :). Went back and forth.